src/models/UserAction.js

```diff
@@ -17,11 +18,13 @@ const userActionSchema = mongoose.Schema({
   },
   timeStamp: {
     type: Date,
-    required: [true, 'Missing timestamp']
+    required: [true, 'Missing timestamp'],
+    default: Date.now
```

Comment:
Date.now is a function, isn't it? https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Global_Objects/Date/now

Did you validate the results in the database? e.g. test this?

Reply:
Yeah it is a function. Mongoose calls it whenever a new UserAction is created without timestamp. We also do this for Offers.

Reply:
So it has been tested and works :)